src/i18n/utils.ts

```diff
@@ -11,6 +11,14 @@ export function useTranslations(lang: keyof typeof ui) {
 	};
 }
 
+export function isOutdated(lang: keyof typeof ui) {
+	if ("version" in ui[lang]) {
+		return ui[lang]["version"] < ui[defaultLang]["version"];
```

Comment:
Hopefully someone more knowledgeable in typescript can provide a cleaner solution, but this fixes type checks:

```suggestion
		return (
			(ui[lang] as (typeof ui)[typeof lang])["version"] <
			ui[defaultLang]["version"]
		);
```